frontend/src/scenes/experimentation/FeatureFlags.js

```diff
@@ -38,6 +40,7 @@ function _FeatureFlags() {
                     </Tooltip>
                 )
             },
+            sorter: (a, b) => moment(a.created_at).isBefore(b.created_at),
```

Comment:
Q: Are these formatted or iso8601 strings? If iso8601 string comparison should also work.